script.js

```diff
@@ -0,0 +1,61 @@
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const addBtn = document.querySelector('.btnAdd');
+const list = document.querySelector('.booklist');
+
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+// function to add books
+function addBook(title, author) {
+  books.push({ title, author });
+  localStorage.setItem('books', JSON.stringify(books));
+  return { title, author };
+}
+
+// function to remove book
+function removeBook(element) {
+  if (element.classList.contains('remove')) {
+    const removeItem = element.parentElement;
+    books.splice(removeItem, 1);
+    removeItem.remove();
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
```

Comment:
- [ ] When I tried to remove elements with diff author but same title, the application deletes the wrong book, please work on it. 🐶 

1. Try to follow the example on images, after deleting with the remove button, refresh the page and you'll see the bug.

![image](https://user-images.githubusercontent.com/20563252/134093602-91d7d029-a620-4c3d-a651-1270da98a33b.png)

Then:

![image](https://user-images.githubusercontent.com/20563252/134093629-7f551915-9bb1-4e9e-b5d4-80db9b60e429.png)

Reply:
This issue is well addressed. Nice job! :clap: